package.json

```diff
@@ -1,23 +1,33 @@
 {
   "name": "polkadot-secure-validator",
-  "version": "3.4.2",
-  "main": "src/index.js",
+  "version": "3.5.0",
   "repository": "https://github.com/w3f/polkadot-secure-validator",
   "author": "W3F Infrastructure Team <devops@web3.foundation>",
   "license": "GPL-3.0",
+  "engines": {
+    "node": "14.16"
```

Comment:
This sets a "required" version of node. Opinions welcome on this, but I have found it is best practice.

CircleCI is using this version. `14.17` is preferred.